freebsd/FreeBSDProcessList.c

```diff
@@ -38,14 +37,17 @@ in the source distribution for its full text.
 #include "Settings.h"
 #include "XUtils.h"
 
+#include "freebsd/FreeBSDMachine.h"
+#include "freebsd/FreeBSDProcess.h"
+
 
 ProcessList* ProcessList_new(Machine* host, Hashtable* pidMatchList) {
```

Comment:
Following along with other platforms this should actually be called `FreeBSDProcessList_new`.

Reply:
@BenBE that makes sense, but its not correct ;)  ...

```
grep ProcessList_new */*ProcessList.c
```